ESISharp/Web/EsiRequest.cs

```diff
@@ -26,35 +26,39 @@ internal EsiRequest(ESIEve EsiObject, string RequestPath)
 
         internal string Get()
         {
-            var Response = GetAsync(RequestUrl);
+            var Response = GetAsync();
             return Response.Result;
         }
 
         internal string Get(object QueryArguments)
         {
-            var ArgString = Utils.ConstructUrlArgs(QueryArguments);
-            var ArgumentRequest = string.Concat(RequestUrl, ArgString);
-            var Response = GetAsync(ArgumentRequest);
+            var Response = GetAsync(QueryArguments);
             return Response.Result;
         }
 
-        private async Task<string> GetAsync(string Url)
+        internal async Task<string> GetAsync(object QueryArguments = null)
```

Comment:
Can you remove the default parameter and overload a parameter-less method? I like to avoid defaults where I can.